.pre-commit-config.yaml

```diff
@@ -8,28 +8,29 @@ repos:
       - id: isort
         name: isort - pd-migrator
         files: ^tools/pagerduty-migrator
-        args:
-          [--settings-file=tools/pagerduty-migrator/.isort.cfg, --filter-files]
+        args: [--settings-file=engine/pyproject.toml, --filter-files]
       - id: isort
         name: isort - dev/scripts
         files: ^dev/scripts
-        args: [--settings-file=dev/scripts/.isort.cfg, --filter-files]
+        args: [--settings-file=engine/pyproject.toml, --filter-files]
 
   - repo: https://github.com/psf/black
-    rev: 22.3.0
+    rev: 23.3.0
     hooks:
       - id: black
         files: ^engine
         args: [--config=engine/pyproject.toml]
       - id: black
         name: black - pd-migrator
         files: ^tools/pagerduty-migrator
+        args: [--config=engine/pyproject.toml]
       - id: black
         name: black - dev/scripts
         files: ^dev/scripts
+        args: [--config=engine/pyproject.toml]
 
   - repo: https://github.com/pycqa/flake8
-    rev: 3.9.2
```

Comment:
bump `flake8` so that we can use `py311` as our target version, which is the actual Python version we use (configured in `engine/pyproject.toml`